client-gateway/bot/driver.py

```
@@ -1,32 +1,371 @@
+            # Store driver info in user_roles
+            user_roles[chat_id] = {
+                'role': 'driver',
+                'registered': True,
+                'driver_id': driver_id,
+                'name': name,
+                'car_description': car_description,
+                'status': 'offline'
+            }
+            
+            logger.info("Driver registered successfully: chat_id=%s driver_id=%s name=%s", 
+                       chat_id, driver_id, name)
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)